### DISCUSSION TITLE
@rstrouse's Pool

### DISCUSSION THREAD

**Discussion by rstrouse** [tagyoureit/nodejs-poolController#406] [My Pool]
36kgal Mediterranean style Pool/Spa combo clad in travertine track suit.  Of course the most important thing is that it I have it connected to njsPC and dashPanel.
IntelliCenter i10PS
2 IntelliFlo VS (old 4x160)
1 Whisperflo Jet Pump
8 IntelliValves
1 Pentair MasterTemp 400k btu gas
1 IntelliChlor IC40
2 Automated Fire Bowls
1 Sheer Descent Waterfall
2 Magicstream Laminars
Clean & Clear 520sf
Filter Pressure Sensor
REM Chem chemistry controller
- IPS Flow Cell
- Whitebox Labs Tentacle T3
- Atlas Scientific Industrial pH probe
- Atlas Scientific Industrial ORP probe
- Altelix Enclosure
- 10 inch touch screen
- Pentair Acid Tank w/Stenner Pump
![image](https://user-images.githubusercontent.com/47839015/162524993-c731c88d-3d9f-4ef8-8144-6ce23bc3cffd.png)

**Comment by rock-crusher** [comment]
Beautiful, I like how the entire pool/spa is raised, simple, clean, yet stunning in its own right, very nice. 👍

**Comment by rstrouse** [comment]
Yeah the idea was inspired by the Fontana di Trevi in Rome.  The idea was that everybody could sit around it and when the wind blows it doesn't get a bunch of crap in it.

**Comment by orrious** [comment]
For your firebowls, did you build or buy your ignition system?

**Reply by rstrouse** [reply]
There is a whole story with the fire bowls but I think these days they are being sold by Grand Effects.

**Reply by orrious** [reply]
I'd like to hear the story or at least enough of it to know if you would use them again or build your own.  

Looking at RV furnace parts, looks like at most $300 in raw parts.  I know these ignition systems go for around $1500+.  What did yours run, if you don't mind me asking.

**Reply by rstrouse** [reply]
These were one of the first consumer models that were built by these guys.  I couldn’t build them for what I bought them for at the time.  The composite bowls were more expensive than the fire inserts.

Today I might build my own since the cost is crazy for these things.  You do want an ignition that is not dependent on an active pilot.  The ignition process should include a pilot that is ignited first then the gas control valve started from a flame sensor.

**Reply by orrious** [reply]
> You do want an ignition that is not dependent on an active pilot. The ignition process should include a pilot that is ignited first then the gas control valve started from a flame sensor.

Correct, no active pilot or no pilot at all.  I've been looking at some of the RV style ignition systems and they don't leverage a pilot at all.  They leverage a two solenoid regulator (both solenoids are ran in parallel off the same signal, assume this is to prevent a fail open scenario), a piezo igniter, and flame sensor.  Just need a control board to open solenoid, run igniter for 10 seconds, sense for flame, if flame ok, else turn off for 5 minutes and try again.

**Reply by rstrouse** [reply]
Mine have the following procedure if the wind blows them out.  Gas shuts off to main valve and almost immediately the ticker starts to re-establish flame.  Within what I would assume the amount of time for the flame sensor to cool enough to kill the main gas valve.

I believe when the on solenoid is engaged the pilot gas valve is open unless it is going through a reset cycle.  This cycle goes like this: If the flame thermocouple does not detect flame on the pilot it will simply begin ticking the ignitor for 20 seconds.  If it does not light the pilot it will shut off the pilot for solenoid for what appears to be around 10 seconds.  I presume this is so the gas concentration can dissipate.  If the gas mixture is too rich it will not ignite the pilot.  It will continue this process in perpetuity until it finally lights and the flame sensor thermocouple senses flame.

When it senses flame then it simply opens the main gas valve.  If at any point the thermocouple opens then it shuts the gas valve off and attempts to relight the pilot.

The design works really well and trouble free.  Even when it is really windy you can run the fire and the controller keeps it lit.

**Reply by orrious** [reply]
Have you ever considered a secondary, much higher flow regulator, with a small secondary ring with larger holes, for the purpose of short intense bursts or is this too far?

**Reply by rstrouse** [reply]
Originally, I contacted the folks who did some of the fire features for the Mirage in Vegas.  I wanted gas fired explosions like those you see on the riverwalk in Melbourne, Australia.  As a young dude I spent way too many hours just sitting and looking at them in amazement when they blew off causing everything on the river to stop.  They randomly go off and cause people on the sidewalk to soil their drawers if they were unaware this could happen.  

For a split second it is like a picture was taken and all things around it stop.  Conversation is muted instantly, If people were walking, they stop mid stride and for a moment it is daylight.   Every, sense is affected right down to the smell of the drawers of the people next to you.  If you couldn't see it you would hear it and if you couldn't hear it you would feel it.  Smell, well that depends on whether you were in a crowd of nervous nellies.

So what you are talking about is what I was looking for.  Adding a ring and more gas is akin to turning the knob on your gas stove.  While you can turn it up you will find that there is a happy fuel/air output that produces the best yellow flame.  I installed 6 inch vents from the bottom of the bowls that draw air up through the flame to give as much flickery flame as you can get.   So think air not gas.  That is unless you want to build a compression canister.  In that case I might suggest isolating it so that nobody can get close enough should something happen.

If this was TL;DR; I think what you are talking about is too far and it won't give you the effect you think it might.

**Reply by orrious** [reply]
I think I can live without huge balls of fire.  Maybe at the next house.  But this does give me something to play with later.  If you're bored, tell us how they do it at the Mirage.

**Reply by rstrouse** [reply]
The bottom line is that they compress the gas to around 50psi and release it into a flame all at once.  But the Mirage isn't impressive like this is.  You have to be sitting there with a drink or knawing on a lamb shank in one of the bars or restaurants to get the full effect.  My friend tells me they now schedule the shows but they used to just randomly go off.  Aussie humor.

From what I understand it is aerosolized LNG under high pressure.

https://www.youtube.com/watch?v=L-VOzZUHxxs

**Reply by rock-crusher** [reply]
Have you seen this one?

![Container Park's iconic praying mantis](https://user-images.githubusercontent.com/94593954/176321072-3e55e4c0-cfb6-49de-b754-b82df2ad0c0c.jpg)

https://youtu.be/mNUseqpOeXc

**Reply by rstrouse** [reply]
That is cool!  Even if I managed to squeak that by the HOA the wife would probably veto it.  I'd probably go with a Godzilla though and have it jam to the famed BOC song.  I went to a BOC concert in the early 80s where the giant fire spitting Godzilla on stage caught fire and the band just kept on playing while the fire department was trying to put it out.  Honestly, I think it was made of wood and paper mache.  Maybe that was really their fire of unknow origin.  Some might think bafoonery but it was epic.

**Reply by orrious** [reply]
Have you opened up your ignition systems to see what model regulator they are using?  Needle in a hay stack trying to find a regulator with pilot and dual solenoids, etc.

Btw, my family is from Geelong.  I get Aussie humour. ;)

**Reply by orrious** [reply]
I think I may have something...  Thoughts?

Honeywell Y8610U4001
https://jacksonsystems.com/product/honeywell-y8610u4001-intermittent-pilot-retrofit-kit/

**Reply by rstrouse** [reply]
I am not so sure that is what you would want.  That has an enclosed pilot assembly and probably not great in the weather.  When I get a chance I will lift out the rocks and flip mine over.  I would assume that the regulator valve is probably fine but the pilot assembly may prove to be a PITA.

**Reply by orrious** [reply]
The pilot assembly is separate on this one.  

This is what comes with it:
![Screenshot from 2022-06-29 16-04-05](https://user-images.githubusercontent.com/9359564/176534924-423cf246-de7d-4cbb-bd18-2679defe46f1.png)

I could also get this one:
https://jacksonsystems.com/product/honeywell-universal-pilot-burner-for-intermittent-pilot-applications-batwing-style-hood/

I will bend a box out of expanded stainless to house it in.  Similar to this:
![pba-10-19](https://user-images.githubusercontent.com/9359564/176534426-77e0ac6c-4795-47f4-b87b-0e109906e82b.jpg)

**Reply by rstrouse** [reply]
Here are a few pics of the fire feature.  Looks like two valves.  One controlled by a thermocouple for the ring and the other controlled by energizing with 24vac for the pilot.  13years and narry a problem.
![image](https://user-images.githubusercontent.com/47839015/176571756-34f4c02b-2951-41ea-9e09-d14b9b5fa5bc.png)
![image](https://user-images.githubusercontent.com/47839015/176571859-e94eaaa3-3f2f-4668-877f-7a6496c33d6f.png)
![image](https://user-images.githubusercontent.com/47839015/176572020-460bef48-44df-41c6-8f81-54660ce85707.png)

**Reply by rock-crusher** [reply]
Hi orrious, I thought about this a few years back and came up with this arrangement. Unfortunately, I never got around experimenting with any of it. It is very inexpensive, simple, and I think it would be fairly easy to implement for use in a (small) fire bowl. No pilot to worry about, just connect to a regulated gas supply and a relay in your pool controller for the 120v glow bar ignitor.

You’re a smart guy, maybe you can figure a way to utilize this type ignition system for your project. 

![Glow Bar Ignition](https://user-images.githubusercontent.com/94593954/176574100-bf6dd979-6040-432b-b7da-a6855b777fe9.JPG)

![Ignitor](https://user-images.githubusercontent.com/94593954/176574127-c9618893-6243-4c25-9616-4660e73ec944.jpg)

![Valve](https://user-images.githubusercontent.com/94593954/176574138-03654bbc-ff30-47ee-ae35-96658d9e7e8a.jpg)

[Ignitor](https://www.amazon.com/Robertshaw-Controls-41-205-NORTON-IGNITOR/dp/B001DHPDEC)
[Gas valve ](https://www.amazon.com/Electrolux-AP2131109-PS446204-Y-30128-35-Krooli/dp/B08BX77HG5/ref=asc_df_B08BX77HG5/?tag=hyprod-20&linkCode=df0&hvadid=459766515995&hvpos=&hvnetw=g&hvrand=1334391126661173062&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9031508&hvtargid=pla-943346703534&psc=1)

**Reply by orrious** [reply]
@rstrouse Can you provide a good pic of the solenoid?  I can't tell the model or manufacturer from the pic.  Or even just the model number..

**Reply by orrious** [reply]
@rock-crusher that's a hot bar style setup.  I don't know how well that would do outside in the elements

**Reply by rstrouse** [reply]
It is an ASCO SV311A01N6CF5 for the main gas valve and ASCO SV311A01N6BF5 for the pilot.

**Reply by orrious** [reply]
They want $150 for each regulator.  The Honeywell regulator I provided above has both main and pilot built into a single unit for $153.  

https://jacksonsystems.com/product/honeywell-vr8345q4563-intermittent-direct-ignition-gas-valve/

Any reason you can think of why I shouldn't pursue this route?

**Reply by rstrouse** [reply]
I am not that skilled at building HVAC systems.  The valve in question appears to be able to handle the gas you will need 60,000 - 80,000BTU for good fire.   It does look a little more difficult to position given it has some exposed solenoid coils and is recommended to be out of the elements.

**Reply by orrious** [reply]
The two smaller units I are for my 30x30 bowls and are good to 200,000 BTU. The larger one, 270,000 BTU, is for my 48" fire pit.  The regulator and control board will be in a metal enclosure that is water proof.    

Inserts seem very reasonable as well.
https://www.amazon.com/Onlyfire-Square-Stainless-Assembly-24-Inch/dp/B01KV2V62C

**Reply by Exit2Studios** [reply]
@rstrouse Do you have a picture of your touch screen installed?  I seem to recall seeing a picture at some point of a touch screen mounted on a hinged piece of plexi.  I would like to implement something similar with the 7" screen I have on the way.

**Comment by rstrouse** [comment]
You mean this.  It is mounted on 3/8" ABS using shower door hinges.  I made some aluminum c-channel supports to mount the hinges for the door.
![image](https://user-images.githubusercontent.com/47839015/180563712-91204260-1f88-426c-be5e-8e5c3737605a.png)

**Reply by Exit2Studios** [reply]
That’s it.  You have pics of the supports?

**Reply by Exit2Studios** [reply]
And why shower door hinges?

**Reply by rstrouse** [reply]
Shower door hinges because they open in a way that allows the door to clear the other door hinges.  The also allow you to simply tighten the hinge onto the ABS as if it was a pane of glass without having to align the door before hooking it up.

Here is a pick when I was building the box.  There is a c-channel on both sides.
![image](https://user-images.githubusercontent.com/47839015/180576233-609a119c-987a-4991-b940-a37c22db5e4b.png)

**Reply by Exit2Studios** [reply]
Thanks for providing that.  I'm getting a "low voltage" warning when connecting my 7" touch screen.  I have the pi powered from the BAS hat, with a more than adequate ps, so not sure what would cause that or what to do to remedy it.  Any suggestions?

**Reply by rstrouse** [reply]
There should be an external power connector on the display.  You will almost never be successful powering a display from the pins on the pi.  I use a buck converter from the 24vdc power supply to the display.

**Comment by rstrouse** [comment]
And here is a pic with the door closed.
![image](https://user-images.githubusercontent.com/47839015/180576365-cd1b491f-f76d-4c5e-a835-31aa9b3daab5.png)

**Reply by orrious** [reply]
That's a nice setup!  I like the door within the door.  I've been on the hunt for a monochrome LCD with 5 buttons that I can put on the outside of the panel but refuse to pay $350+ for one.

**Comment by rstrouse** [comment]
Elecrow 10.5 inch. It was less than $100 when I bought it.  Now it’s $170.
